### PR TITLE
Make first draft of content changes

### PR DESCRIPTION
**Pull request by tsmith023**
### Type of change:

- [ ] **Documentation** updates (non-breaking change to fix/update documentation)
- [ ] **Website** updates (non-breaking change to update main page, company pages, pricing, etc)
- [x] **Content** updates – **blog**, **podcast** (non-breaking change to add/update content)
- [ ] **Bug fix** (non-breaking change to fixes an issue with the site)
- [ ] **Feature** or **enhancements** (non-breaking change to add functionality)

### How Has This Been Tested?

- [ ] **Github action** – automated build completed without errors
- [x] **Local build** - the site works as expected when running `yarn start`

> note, you can run `yarn verify-links` to test site links locally

### CODE REVIEW

File: blog/2024-03-20-typescript-client-beta/index.mdx [hunk 1]
```diff
@@ -89,25 +89,27 @@ response.generated  // Get the generated text from a `groupedTask` request
...
+### Separated Node and Web Versions
 
-For a long time, we've used a single client to intereact with Weaviate from both the Web and Node.js. With the new client we've abstracted both environments into two bundles accessible in the same package. 
+For a long time, we've used a single client to intereact with Weaviate from both the Web and Node.js. This made sense in the world of REST and GraphQL, which are both HTTP/1.1 protocols, but with the advent of gRPC into Weaviate since `1.24`, we now have the HTTP/2 protocol to contend with. As it transpires, gRPC and HTTP/2 don't play nicely with browsers so we've chosen to break up the new TypeScript client into Node and Web versions.
 
-- `weaviate-client/node`
-- `weaviate-client/web`
+During this open community-driven beta, you'll only have access to the Node version that can be accessed from `weaviate-client`:
+```typescript
+import weaviate from 'weaviate-client';
```
Comment (malgamves) [inline blog/2024-03-20-typescript-client-beta/index.mdx:98]: should this be `import weaviate from 'weaviate-client/node` ?

File: blog/2024-03-20-typescript-client-beta/index.mdx [hunk 2]
```diff
@@ -328,146 +331,71 @@ We won't get into the details of gRPC here, but if you are interested, you can r
...
 ```typescript
-import weaviate.classes as wvc  # <-- import the custom classes
-```
+import weaviate, { Collection } from 'weaviate-client';
 
-[whats the equivalent? Default exports?]
+const collection: Collection = client.collections.get("MyCollection");
+```
 
-### Collection definition
```
Comment (malgamves) [inline blog/2024-03-20-typescript-client-beta/index.mdx:345]: you completely took out the collections definition section, don't think we should have it?

Comment (tsmith023) [reply blog/2024-03-20-typescript-client-beta/index.mdx:345]: Ah, my mistake! I meant to refactor and add it back but didn't before opening the PR. Will add now 😁

Comment (malgamves) [reply blog/2024-03-20-typescript-client-beta/index.mdx:345]: ok, cool!

**Pull request merged**